cogboard-webapp/src/components/SnackbarWithVariant.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import CheckCircleIcon from '@material-ui/icons/CheckCircle';
+import ErrorIcon from '@material-ui/icons/Error';
+import InfoIcon from '@material-ui/icons/Info';
+import CloseIcon from '@material-ui/icons/Close';
+import WarningIcon from '@material-ui/icons/Warning';
+import {amber, blue, green, red} from '@material-ui/core/colors';
+import IconButton from '@material-ui/core/IconButton';
+import Snackbar from '@material-ui/core/Snackbar';
+import SnackbarContent from '@material-ui/core/SnackbarContent';
+import {makeStyles} from '@material-ui/core/styles';
+
+const useVariantStyles = makeStyles(theme => ({
```

Comment:
Please stick to styled components approach. You can map status to color writing some helper function like in Widget component. Check it for reference. If not already defined, please define this colors in theme.js file.

Reply:
Fixed.